package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "brainstem-js",
+  "repository": "https://github.com/mavenlink/brainstem-js",
+  "filename": "brainstem",
+  "standalone": "Brainstem",
+  "version": "0.4.0-2",
+  "author": {
+    "name": "Mavenlink",
+    "email": "dev@mavenlink.com"
+  },
+  "license": "MIT",
+  "main": "lib/brainstem.js",
+  "scripts": {
+    "prepublish": "gulp build-module",
+    "test": "gulp test"
+  },
+  "dependencies": {
+    "backbone": "^1.1.2",
+    "jquery": "^2.1.4",
+    "underscore": "^1.4.4"
+  },
+  "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-core": "^6.7.2",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "backbone-factory": "https://github.com/jrolfs/Backbone-Factory.git#master",
+    "browserify": "^13.0.0",
+    "browserify-shim": "^3.8.12",
+    "coffeeify": "^2.0.1",
+    "del": "^2.2.0",
+    "gulp": "^3.9.1",
+    "gulp-coffee": "^2.3.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-replace": "^0.5.4",
+    "gulp-streamify": "^1.0.2",
+    "gulp-uglify": "^1.5.3",
+    "gulp-util": "^3.0.7",
+    "inflection": "^1.8.0",
+    "jasmine-core": "^2.4.1",
+    "jasmine-jquery-matchers": "^1.1.1",
+    "karma": "^0.13.22",
+    "karma-browserify": "^5.0.2",
+    "karma-chrome-launcher": "^0.2.2",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-jasmine": "^0.3.8",
+    "karma-sinon": "^1.0.4",
+    "karma-spec-reporter": "0.0.24",
+    "minimist": "^1.2.0",
+    "path": "^0.12.7",
+    "sinon": "^1.4.1",
+    "vinyl-source-stream": "^1.1.0",
+    "watchify": "^3.7.0"
+  },
+  "browserify-shim": {
+    "jquery": "global:$",
+    "underscore": "global:_",
+    "backbone": "global:Backbone"
```

Comment:
No, this is necessary for the bundled gem output to prevent bundling them as requirements:
https://github.com/mavenlink/brainstem-js/blob/common-js/gulpfile.babel.js#L39

Ideally we can find a way to pass this configuration directly into the `shim` transform in the Gulpfile but I believe I tried that and it didn't work.

Something like this:
```javascript

gulp.task('build-gem', () => {
  const shim = shim({
    jquery: 'global:$',
    underscore: 'global:_',
    backbone: 'global:Backbone'
  });

  return browserify(source, {
    standalone,
    transform: [coffeeify, shim],
    extensions: ['.coffee']
  }).bundle()
  // ...
});
```